### ISSUE TITLE
maybe bug with bottle filler

### ISSUE THREAD

**Issue by Nabbs1** [Alesseon/ONI-BottlesToPipes#8]
Bottle emptier works awesome.  Bottle filler doesnt auto disperse and dupes dont get contents from it.

On the DLC with water it works but only if I click the building and eject contents. Then a bottle appears and dupe can come pick it up. Without the eject contents nothing gets bottled water

**Comment by Nabbs1** [comment]
Empty Storage I meant

**Comment by Alesseon** [comment]
"Nabbs1" Back to report! Its not a bug at all and the mod works great. wanted to share that OtherMultiplier in the mod "Bigger Capacity" causes the bottlers to break if set higher than default (1.0).

**Comment by Nabbs1** [comment]
Yeah I figured that out .. should of posted it here.  I did post it on the steam forum for the addon tho so no one else would report same.

**Comment by Alesseon** [comment]
Hi, i was just copying your message word for word so at least i have there what mod is conflicted. :)

**Comment by Alesseon** [comment]
Maybe second mod incompatibility.
The dupes aren't auto taking from the dispenser for me, only other mods I have are More Food Variety and DGSM (Dupe Gen Setting Manager). I am playing spaced out DLC, not sure if that is an issue with the most recent update.
